dev/tests/integration/testsuite/Magento/Framework/View/Layout/Reader/BlockTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Magento\Framework\View\Layout\Reader;
+
+class BlockTest extends \PHPUnit_Framework_TestCase
+{
+    const IDX_TYPE = 0;
+    const IDX_PARENT = 2;
+
+    /**
+     * @var Block
+     */
+    private $block;
+
+    /**
+     * @var Context
+     */
+    private $readerContext;
+
+    private $blockName = 'test.block';
+    private $childBlockName = 'test.child.block';
+
+    public function setUp()
+    {
+        $this->block = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->create(
+            \Magento\Framework\View\Layout\Reader\Block::class
```

Comment:
Replace `\Magento\Framework\View\Layout\Reader\Block::class` with `"\Magento\Framework\View\Layout\Reader\Block"`
All same usage in test will be refactored automatically after php 5.4 become deprecated.